### PR TITLE
modern-acad-cv:0.1.0

### PR DESCRIPTION
**Pull request by bpkleer**
I am submitting
- [x] a new package/template
- [ ] an update for a package

Description: This template for an academic CV serves the peculiarities of academic CVs. If you are not an academic, this template is not useful. Most of the times in academics, applicants need to show everything they have done. This makes it a bit cumbersome doing it by single entries. In addition, academics might apply to institutions around the globe, making it necessary to send translated CVs or at least translations of some parts (i.e., title of papers in different languages). 

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE
- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/modern-acad-cv/0.1.0/template/example.typ [hunk 1]
```diff
@@ -0,0 +1,109 @@
+// #import "modern-acad-cv.typ": *
+#import "@local/modern-acad-cv:0.1.0": *
```
Comment (elegaanz) [inline packages/preview/modern-acad-cv/0.1.0/template/example.typ:2]: Could you use the `@preview` import here? This file will be copied as is when creating a project from your template, and these imports won't work by default.

```suggestion
#import "@preview/modern-acad-cv:0.1.0": *
```

File: packages/preview/modern-acad-cv/0.1.0/typst.toml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+compiler = "0.11.0"
+entrypoint = "lib.typ"
+authors = ["bpkleer (Philipp Kleer) <philipp.kleer@posteo.com>"]
+license = "MIT"
+description = "A CV template for academics based on moderncv LaTeX package."
+keywords = ["CV", "Curriculum Vitae", "Resume", "Academics", "Professional", "Modern"]
+categories = ["cv"]
+disciplines = []
+homepage = "https://github.com/bpkleer/modern-acad-cv"
+repository = "https://github.com/pavelzw/modern-acad-cv"
```
Comment (elegaanz) [inline packages/preview/modern-acad-cv/0.1.0/typst.toml:13]: None of these links seem to work, can you please fix them or remove them?

File: packages/preview/modern-acad-cv/0.1.0/template/example.typ (hunk 1 above)
Comment (bpkleer) [reply packages/preview/modern-acad-cv/0.1.0/template/example.typ:2]: Ah thanks for the quick check. I'll update and resend.

**Comment by elegaanz** [comment]
Thank you!

**Pull request merged**